docs/runtimes/images.md

```diff
@@ -24,16 +24,22 @@ All images are published to
 The images are:
 
 - `mlrun/mlrun`: An MLRun image includes preinstalled OpenMPI and other ML packages. Useful as a base image for simple jobs. 
+- `mlrun/mlrun-kfp`: The same as `mlrun/mlrun` with the addition of KFP 1.8 already installed. **This image includes KFP.**
```

Comment:
it is quiet the same. not exactly. `mlrun/mlrun`, comes with pydantic v2 while mlrun-kfp comes with pydantic v1 oob. the gap is because kfp 1.8 dependency comes with more restrictions.
perhaps we can refine the text to say
```suggestion
- `mlrun/mlrun-kfp`: MLRun based image with KFP 1.8 preinstalled. **This image includes KFP.** and being used for compiling kfp workflows, for remote workflows.
```